libr/include/r_util/r_uleb128.h

```diff
@@ -5,7 +5,7 @@
 // compression used to store an arbitrarily large integer in a small number of
 // bytes. LEB128 is used in the DWARF debug file format.
 
-R_API const ut8 *r_uleb128(const ut8 *data, int datalen, ut64 *v);
+R_API const ut8 *r_uleb128(const ut8 *data, int datalen, ut64 *v, bool *success);
```

Comment:
if its not successful can we return NULL and avoid changing the signature i think.

Reply:
this will break the concept of caller calling eprintf with the relevant error message (thus, will break ci)

Reply:
and I cannot guarantee that none of the other callers to r_uleb128 aren't counting on valid return value even if decoding fails.